src/test/java/org/folio/dew/bursarfeesfines/NoFeesFinesTest.java

```diff
@@ -37,7 +38,7 @@
 import org.springframework.batch.test.JobLauncherTestUtils;
 import org.springframework.beans.factory.annotation.Autowired;
 
-class BursarFeesFinesTest extends BaseBatchTest {
+class NoFeesFinesTest extends BaseBatchTest {
 
   private static final String USERS_ENDPOINT_PATH = "/users";
   private static final String ITEMS_ENDPOINT_PATH = "/inventory/items";
```

Comment:
Can't add comments outside of the changed lines.

lines 56/57 `bursarExportService` is not used and it can be removed.

lines 157/158 we don't want to use random values for tests; hardcode in a UUID (random or something like literal `"00000000-0000-4000-8000-000000000000"`)